pkgs/shells/fish/wrapper.nix

```diff
@@ -1,19 +1,43 @@
-{ lib, writeShellScriptBin, fish }:
+{ lib, writeShellScriptBin, fish, writeTextFile }:
 
 with lib;
 
```

Comment:
```suggestion
```
with shouldn't be used over the entire file like this.

Reply:
What do you suggest? To completely remove it?

Reply:
Here you go, I removed it. I hope it's ok.